modules/ossm-about-directing-egress-traffic-through-a-gateway.adoc

```diff
@@ -5,4 +5,8 @@
 [id="ossm-about-directing-egress-traffic-through-a-gateway_{context}"]
 = About directing egress traffic through a gateway
 
-You can configure a gateway to define exit points from a mesh. This allows you to apply {istio} features, such as monitoring and route rules, to the traffic exiting the mesh.
\ No newline at end of file
+You can configure a gateway installed using gateway injection as an exit point for the traffic leaving a service mesh. In this configuration, the gateway acts as a forward proxy for requests sent to services that are external to the mesh. 
+
+Configuring a gateway for egress traffic helps fulfill security requirements. For example, if traffic restrictions in an environment require that the traffic exiting a mesh flows through a dedicated set of nodes. Similarly, when network policies prevent the nodes on which applications run from directly accessing external services. In such cases, the gateway proxies run on dedicated egress nodes that are capable of accessing external services. The traffic on these nodes may be subject to strict network policy enforcement or additional monitoring.
```

Comment:
"For example, if..." is not a complete sentence. Should this be part of the previous sentence as an example of configuring a gateway fulfilling security requirements?

LIkewise with "similarly, when network policies..." -- this is not a complete sentence.

Reply:
revised paragraph and will touch base with dev to confirm changes from technical standpoint.